packages/@react-aria/menu/src/useMenuTrigger.ts

```diff
@@ -67,10 +72,10 @@ export function useMenuTrigger(props: MenuTriggerAriaProps, state: MenuTriggerSt
       id: menuTriggerId,
       onPressStart(e) {
         // For consistency with native, open the menu on mouse/key down, but touch up.
-        if (e.pointerType !== 'touch') {
-          // If opened with a keyboard or screen reader, auto focus the first item.
+        if (e.pointerType !== 'touch' && e.pointerType !== 'keyboard') {
```

Comment:
I don't think there is a particular reason that keyboard handling needs to be handled in onPressStart rather than in the onKeyDown above, but lemme know if I overlooked something or am missing a keyboard interaction that should open the menu other that the ones covered by onKeyDown above

Reply:
Is it possible to trigger a press event on mobile for accessibility via a keyboard or voiceover? I agree that onKeyDown should handle that, but wondering if this is to prevent this from getting called twice.

Reply:
yeah, for mobile onKeyDown will handle the toggling from keyboard events (e.g. a keyboard connected to a phone) and the press events get handled by `onPress` below so I think we are fine here